### PR TITLE
Add missing unit information to amounts

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR adds missing unit information to amounts for clarity.

* Use singular unit where possible, i.e., Sats -> Sat
* Remove unused `AccountState.onChainFeeRate`
* Fix typos on `PaymentError`'s

### CODE REVIEW

File: lib/routes/lnurl/payment/lnurl_payment_page.dart [hunk 1]
```diff
@@ -226,46 +226,54 @@ class LNURLPaymentPageState extends State<LNURLPaymentPage> {
...
   }
 
-  String? validatePayment(int amount) {
+  String? validatePayment(int amountSat) {
     final texts = context.texts();
     final accBloc = context.read<AccountBloc>();
     final lspState = context.read<LSPBloc>().state;
     final currencyState = context.read<CurrencyBloc>().state;
 
     final maxSendable = widget.data.maxSendable ~/ 1000;
```
Comment (dangeross) [inline lib/routes/lnurl/payment/lnurl_payment_page.dart:256]: maxSendableSat ?

File: lib/routes/lnurl/payment/lnurl_payment_page.dart [hunk 2]
```diff
@@ -226,46 +226,54 @@ class LNURLPaymentPageState extends State<LNURLPaymentPage> {
...
-    if (amount > maxSendable) {
+    if (amountSat > maxSendable) {
       return texts.lnurl_payment_page_error_exceeds_limit(maxSendable);
     }
 
     final minSendable = widget.data.minSendable ~/ 1000;
```
Comment (dangeross) [inline lib/routes/lnurl/payment/lnurl_payment_page.dart:261]: minSendableSat ?

File: lib/routes/lnurl/payment/lnurl_payment_page.dart (hunk 1 above)
Comment (erdemyerebasmaz) [reply lib/routes/lnurl/payment/lnurl_payment_page.dart]: Changed!

File: lib/routes/lnurl/payment/lnurl_payment_page.dart (hunk 2 above)
Comment (erdemyerebasmaz) [reply lib/routes/lnurl/payment/lnurl_payment_page.dart]: Changed!

**Review by dangeross — APPROVED** [review]
LGTM

**Pull request merged**